_posts/2020-09-14-Introduction-to-API-Management.md

```diff
@@ -0,0 +1,292 @@
+---
+layout: post
+authors: [Kris_Jordens]
+title: 'Introduction to API Management'
+image: /img/kris_Jordens.jpg
+tags: [API Management, API Gateway, Products, Responsibilities]
+category: API
+comments: true
+---
+
+        
+# Introduction to API Management
+
+
+## Table of contents
+* [Introduction to API Management](#Introduction-to-API-Management)
+    * [Problem description](#Problem-description)
+    * [Investigation](#Investigation)
+        * [API Management or API Gateway?](#API-Management-or-API-Gateway)
+        * [API landscape without an API management tool](#API-landscape-without-an-API-management-tool)
+        * [API landscape with an API management tool](#API-landscape-with-an-API-management-tool)
+        * [Responsibilities of an API Management tool](#Responsibilities-of-an-API-Management-tool)
+        * [Challenges](#Challenges)
+            * [Multiple API gateways configured](#Multiple-API-gateways-configured)
+            * [Clustering gateways](#Clustering-gateways)            
+    * [API management products](#API-management-products) 
+    * [Hands-on](#Hands-on)
+        * [Installation](#Installation)
+        * [Setup](#Setup)
+        * [Automation](#Automation)
+    * [Conclusion](#Conclusion)
+
+
+## Problem description
+
+In our current digital world where everyone is creating APIs, we should look at how we can manage and control them.
+
+
+Most of us know that we not only have to build the API but foresee additional responsibilities. 
+Some are just a necessary evil, like adding cross-cutting concerns.  
+Keeping an overview of all available API's can become a struggle these days.
+
+
+        
+Well, instead of trying to reinvent the wheel, I searched for the best solution and found that an API Management tool can solve a lot of those problems and add extra value within your landscape.
+
+An API management tool helps you in managing your APIs, setting the responsibilities on the right level. 
+
+
+You can add plenty additional functionalities, but most of all this happens in one tool!
+That’s one of the reasons it is important for your business to implement such a tool in your organization.
+        
+                
+## Investigation
+
+### API Management or API Gateway?
+The terms API management and API gateway are often used interchangeably, which is not correct.
+An API management tool refers to the application or overall solution of managing APIs.

```

Comment:
API Gateway is the main component of API management solutions.
an API management solution consists of more components: API Gateway, API portal, API Governance

Reply:
Hi Kevin,
What do you mean, or what change would you suggest with this comment?

I do think that it can but doesn't have to be a SAASs solution. 
Certain API Management applications can be hosted and managed 'on premise' by the customer.

Reply:
look at the image of this site: https://www.softwaretestinghelp.com/api-management-tools/
I think you want to mean that an API Management tool is a set of components, including the API gateway no which can be deployed on-premise or use of SAAS solutions

Reply:
That's exactly what I meant .
I rephrased this section a bit.
I didn't include the on-premise or Saas solution part, as this section is actually about the naming's.
can you let me know if it's more clear , or if I you have some other suggestions?